controllers/users/users.js

```diff
@@ -17,14 +18,12 @@ exports.registerUser = (req, res) => {
 
   User.findOne({ email: email.toLowerCase() }).then(user => {
     if (user) {
-      return res
-        .status(400)
-        .json({ errors: [{ msg: "Email already exists" }] });
+      return res.status(400).json({ errors: [{ msg: EAE }] });
```

Comment:
EAE say nothing to me. Please rephrase it to EMAIL_ALREADY_EXISTS or something like that